ta/pkcs11/src/pkcs11_helpers.c

```diff
@@ -278,6 +278,9 @@ static const struct any_id __maybe_unused string_rc[] = {
 	PKCS11_ID(PKCS11_CKR_KEY_INDIGESTIBLE),
 	PKCS11_ID(PKCS11_CKR_SESSION_READ_ONLY_EXISTS),
 	PKCS11_ID(PKCS11_CKR_SIGNATURE_LEN_RANGE),
+	PKCS11_ID(PKCS11_CKR_UNWRAPPING_KEY_HANDLE_INVALID),
+	PKCS11_ID(PKCS11_CKR_UNWRAPPING_KEY_SIZE_RANGE),
+	PKCS11_ID(PKCS11_CKR_UNWRAPPING_KEY_TYPE_INCONSISTENT),
```

Comment:
@ruchi393: please add following too here:
```
	PKCS11_CKR_WRAPPED_KEY_INVALID		= 0x0110,
	PKCS11_CKR_WRAPPED_KEY_LEN_RANGE	= 0x0112,
	PKCS11_CKR_WRAPPING_KEY_HANDLE_INVALID  = 0x0113,
	PKCS11_CKR_WRAPPING_KEY_SIZE_RANGE	= 0x0114,
	PKCS11_CKR_WRAPPING_KEY_TYPE_INCONSISTENT = 0x0115,
	PKCS11_CKR_RANDOM_SEED_NOT_SUPPORTED	= 0x0120,
	PKCS11_CKR_RANDOM_NO_RNG		= 0x0121,
```

Reply:
Sure, I will fix it. Sorry for missing it the second time :-(